### PR TITLE
Aqara Z1 improvement

### PR DESCRIPTION
**Pull request by waterlou**
- use binary instead of enum for lock relay state
- use boolean instead of string for lock relay state
- decode lock relay payload

### CODE REVIEW

**Comment by waterlou** [comment]
add led_disabled_night for z1 switch

File: src/lib/lumi.ts [hunk 1]
```diff
@@ -1659,25 +1663,25 @@ export const toZigbee = {
...
                 break;
             case 'serving_size':
                 // @ts-expect-error
-                await sendAttr(0x0e5c0055, value, 4);
+                sendAttr(0x0e5c0055, value, 4);
                 break;
             case 'portion_weight':
                 // @ts-expect-error
-                await sendAttr(0x0e5f0055, value, 4);
+                sendAttr(0x0e5f0055, value, 4);
```
Comment (Koenkk) [inline src/lib/lumi.ts:1684]: Please keep all `await`

Comment (waterlou) [reply src/lib/lumi.ts]: fixed

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**